compiler-core/src/analyse/imports.rs

```diff
@@ -248,8 +259,12 @@ impl<'context, 'problems> Importer<'context, 'problems> {
                 .insert(used_name.clone(), (import.location, import_info));
 
             self.environment
-                .value_names
-                .imported_module(import.module.clone(), used_name)
+                .names
+                .imported_module(import.module.clone(), used_name.clone());
+
+            self.environment
+                .names
+                .imported_module(import.module.clone(), used_name);
```

Comment:
Is this intentionally here twice?

Reply:
No, that must have been left over from the merger of `TypeNames` and `ValueNames`